### PR TITLE
Reanimate travis build

### PR DESCRIPTION
**Pull request by aik099**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+      dist: trusty
+    - php: 5.5
+      dist: trusty
+    - php: 5.3
+      dist: precise
 
 env:
   global:
     # Force using PHP 5.6 for the webserver to be able to run it on PHP 5.3 and HHVM jobs too
     - MINK_PHP_BIN=~/.phpenv/versions/5.6/bin/php
```
Comment (stof) [inline .travis.yml]: I suggest doing that only in the `5.3` job instead.

**Comment by aik099** [comment]
I've noticed 2 things failing in the build:

1. on PHP 5.3 curl doesn't connect to http://localhost:8002/basic_auth.php , which suggests that local web server might be started on different port OR not started at all due absence of PHP 5.6
2. on PHP 5.4 two tests are failing, but I see them working locally on local web server (not test php server from php binary)

@stof, ideas?

**Comment by stof** [comment]
for the first point, redirect the output of the mink test server script to a file rather than to `/dev/null`, and display the cotnent on this file after failures (see the Selenium2Driver config, it does it). This will let you know what the issue is.

According to https://travis-ci.org/github/minkphp/MinkGoutteDriver/jobs/661443721#L88, maybe we need to use `5.6.13` rather than `5.6` to reference the preinstalled PHP 5.6 version.

**Comment by stof** [comment]
The failure for `Behat\Mink\Tests\Driver\Form\Html5Test::testHtml5FormAction` is due to the version of symfony/dom-crawler being used. Support for `formaction` on buttons was implemented in 3.3 but this jobs installs 2.8. The test should be skipped (see the logic in BrowserKitDriver).

The same is true for `formmethod` btw.

**Comment by aik099** [comment]
You're correct @stof . Now everything works. Thank you.

**Pull request merged**